packages/mocaccino-dracut/build.yaml

```diff
@@ -0,0 +1,14 @@
+image: "quay.io/mocaccino/desktop:master"
```

Comment:
Please use the micro imagehttps://github.com/mocaccinoOS/mocaccino-extra/blob/ddcc7ba83b06c1dd6a8c7648509233c4bd864977/packages/keybase-tools/build.yaml#L1 , or just require the toolchain as in https://github.com/mocaccinoOS/mocaccino-extra/blob/ddcc7ba83b06c1dd6a8c7648509233c4bd864977/packages/e2fsprogs/build.yaml#L2 . If you do the latter, you can drop the `image` keyword entirely

Reply:
The image is just used for building, does have no effect while installing the package